app/api/module/Api/src/Entity/Si/SiPenalty.php

```diff
@@ -29,28 +30,31 @@ class SiPenalty extends AbstractSiPenalty
     public function __construct(
         SiEntity $seriousInfringement,
         SiPenaltyTypeEntity $siPenaltyType,
+        SiPenaltyErruRequestedEntity $requestedErru,
         $imposed,
         \DateTime $startDate = null,
         \DateTime $endDate = null,
         $reasonNotImposed = null
     ) {
         $this->seriousInfringement = $seriousInfringement;
-        $this->update($siPenaltyType, $imposed, $startDate, $endDate, $reasonNotImposed);
+        $this->update($siPenaltyType, $requestedErru, $imposed, $startDate, $endDate, $reasonNotImposed);
     }
 
     /**
      * @param string $imposed
-     * @param string $reasonNotImposed
+     * @param string $reasonNotImposedz
```

Comment:
Typo here. If these are required fields (are they? But guessing they are...) you could also put proper string type hints into the PHP code and that would allow for removal of the old docblocks.